tests/test_ble_interface_fixtures.py

```
@@ -583,14 +677,21 @@ def _build_interface(
+        no-op, `_connect_stub_calls` recording addresses passed to the stubbed
+        `connect`, and `_connect_stub_kwargs` recording keyword arguments.
     """
     ble_mod = _get_ble_module()
-    BleInterfaceClass = ble_mod.BLEInterface
+    BleInterfaceClass = cast(type["BLEInterface"], ble_mod.BLEInterface)
     connect_calls: list[str | None] = []
+    connect_call_kwargs: list[dict[str, object]] = []
 
     def _stub_connect(
-        _self: Any, _address: str | None = None, *args, **kwargs
+        _self: Any,
```

Ruff (0.15.4)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_self`

(ANN401)